internal/cmd/operator-sdk/pkgManToBundle/cmd.go

```diff
@@ -0,0 +1,315 @@
+
+The generated bundles are always written on disk. Location for the generated bundles can be specified using '--output-dir'. If not
+specified, the default location would be 'bundle/' directory.
+
+The base container image name for the bundles can be provided using '--base-image' flag. This should be provided without the tag, since the tag
+for the images would be the bundle version, (ie) image names will be in the format <base_image>:<bundle_version>.
+
+Specify the build command for building container images using '--build-cmd' flag. The default build command is 'docker build'. The command will
+need to be in the 'PATH' or fully qualified path name should be provided.
+`
+
+	examples = `
+
+# Provide the packagemanifests directory as input to the command. Consider the packagemanifests directory to have the following
+# structure:
+
+$ tree packagemanifests/
+packagemanifests
+└── etcd
+    ├── 0.0.1
+    │   ├── etcdcluster.crd.yaml
+    │   └── etcdoperator.clusterserviceversion.yaml
+    ├── 0.0.2
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.0.2.clusterserviceversion.yaml
+    │   └── etcdrestore.crd.yaml
+    └── etcd.package.yaml
+
+# Run the following command to generate bundles in the default 'bundle/' directory with the base-container image name
+# to be 'quay.io/example/etcd'
+$ operator-sdk pkgman-to-bundle packagemanifests --base-image quay.io/example/etcd
+INFO[0000] packagemanifests will be migrated to bundles in bundle directory
+INFO[0000] Creating bundle/bundle-0.0.1/bundle.Dockerfile
+INFO[0000] Creating bundle/bundle-0.0.1/metadata/annotations.yaml
+...
+
+# After running the above command, the bundles will be generated in 'bundle/' directory.
+$ tree bundle/
+bundle/
+├── bundle-0.0.1
+│   ├── bundle.Dockerfile
+│   ├── manifests
+│   │   ├── etcdcluster.crd.yaml
+│   │   ├── etcdoperator.clusterserviceversion.yaml
+│   ├── metadata
+│   │   └── annotations.yaml
+│   └── tests
+│       └── scorecard
+│           └── config.yaml
+└── bundle-0.0.2
+    ├── bundle.Dockerfile
+    ├── manifests
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.0.2.clusterserviceversion.yaml
+    │   ├── etcdrestore.crd.yaml
+    ├── metadata
+    │   └── annotations.yaml
+    └── tests
+        └── scorecard
+			└── config.yaml
+
+Also, images for the both the bundles will be built with the following names: quay.io/example/etcd:0.0.1 and quay.io/example/etcd:0.0.2.
+`
+)
+
+type pkgManToBundleCmd struct {
+	// Input packagemanifest directory.
+	pkgmanifestDir string
+
+	// Optional flags for generating and building bundles.
+	outputDir string
+	baseImg   string
+	buildCmd  string
+}
+
+// NewCmd returns the pkgManToBundleCmd configured with the provided input options.
+func NewCmd() *cobra.Command {
+	p := pkgManToBundleCmd{}
+
+	pkgManToBundleCmd := &cobra.Command{
+		Use:     "pkgman-to-bundle",
```

Comment:
```suggestion
		Use:     "pkgman-to-bundle <packagemanifestdir>",
```

Should include the required parameter. https://github.com/operator-framework/enhancements/blob/master/enhancements/sdk-migrate-bundle.md#command-line-interface